bot/exts/info/pypi.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from random import choice
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+URL = "https://pypi.org/pypi/{package}/json"
+FIELDS = ["author", "requires_python", "summary", "license"]
+
+log = logging.getLogger(__name__)
+
+
+class PyPi(Cog):
+    """Cog for getting information about PyPi packages."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @command(name="pypi", aliases=("package", "pack"))
```

Comment:
Would `pip` be another good alias, or do we want to reserve that for a future tag?

Reply:
We may have a tag for that at some point, so I think we should reverse it for future use.